src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = days * 40;
+
+  if (days >= 7) {
+    return rentalCost - 50;
+  }
+
+  if (days >= 3) {
+    return rentalCost - 20;
+  }
+
+  return rentalCost;
```

Comment:
Prefer using 'const' over 'let' for 'rentalCost' as its value is not being reassigned, which helps to prevent unintentional changes.